.gitlab/tests/nightly.yml

```diff
@@ -56,10 +62,16 @@ include:
           - hpl
           - kripke
           - lammps
-          - qws
           - raja-perf
+        VARIANT: [+openmp]
+      - HOST: dane
+        ARCHCONFIG: llnl-cluster
+        SCHEDULER_PARAMETERS: $DANE_PARAMS
+        BENCHMARK:
+          - qws
           - salmon-tddft
         VARIANT: [+openmp]
+        SYSTEM_ARGS: [compiler=gcc]
```

Comment:
@rfhaque Is the appropriate solution to fix these for oneapi instead of using gcc, or use some other compiler?